shared/constants/hardware-wallets.ts

```diff
@@ -16,7 +16,7 @@ export enum HardwareKeyringNames {
 }
 
 export enum HardwareDeviceNames {
-  ledger = 'ledger',
+  ledger = 'Ledger',
```

Comment:
I found that this fixes the issue around only deriving one account:

https://github.com/user-attachments/assets/24bef28c-b2e7-4b2c-86b5-8cf2a2f4e0a6